content/dictionary/fontface.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: fontface
-definition: ''
-perspectives: []
+definition: FontFace is a CSS rule that displays text with different custom fonts.
+sources: 
+- sourceurl: https://developer.mozilla.org/en-US/docs/Web/API/FontFace
+perspectives: 
+- meaning: Fontfaces allow fonts to be stored either locally or remotely. When it happens to be loaded can be controlled to.
```

Comment:
```suggestion
definition: FontFace is a CSS rule that displays text with different custom fonts. Fontfaces allow fonts to be stored either locally or remotely. When it happens to be loaded can be controlled to.
sources: 
- sourceurl: https://developer.mozilla.org/en-US/docs/Web/API/FontFace
perspectives: 
```
Since the added information is applicable to the entire definition and not just one role, I'd suggest adding it to the definition.